.gitignore

```diff
@@ -30,6 +30,7 @@
 # Auto generated headers
 /crypto/buildinf.h
 /include/crypto/*_conf.h
+/include/internal/param_names.h
```

Comment:
This one has been removed and shouldn't be needed.
You'll get it if you switch to 3.5 or before and then back to master but it won't be generated from master.

Reply:
Fixed, thanks